.github/workflows/test.yaml

```diff
@@ -20,8 +20,11 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
+      - name: build images
+        working-directory: .
+        run: ./build-docker-images.sh devnet
```

Comment:
[Nit] Maybe we can add a npm script for building images so that we can pass in the version from `package.json` so we do not need to update version manually inside the `build-docker-images.sh`

Reply:
Sounds great, will update!